test/a-src.js

```diff
@@ -59,9 +65,25 @@ describe('src', function () {
 	it('should accept file objects', function (done) {
 		vfsFake.src(fakeFiles[0])
 			.pipe(through.obj(function (file) {
-				file.path.should.equal(fakeFiles[0].path);
-				file.contents.should.equal(fakeFiles[0].contents);
+				file.path.should.equal(fakePath);
+				file.contents.toString().should.equal(fakeContent);
+				file.base.should.equal(process.cwd());
+				file.cwd.should.equal(process.cwd());
+				done();
+			}));
+	});
+
+	it('should accept file objects with no content', function (done) {
+		vfsFake.src({
+			path: fakePath
+		})
+			.pipe(through.obj(function (file) {
+				file.path.should.equal(fakePath);
 
+				// can't use .should.equal on null!
+				assert.equal(file.contents, null);
```

Comment:
You can use the `should` module for this still! `should.equal(file.contents, null)` is just aliased to `assert.equal`, I think, and it would mean you don't have to load the assert library.